### PR TITLE
bowfile.com

### PR DESCRIPTION
**Pull request by piquark6046**
### To avoid invalid pull requests, please check and confirm following terms

- [X] This is not an ad/bug report;
- [X] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [X] I have performed a self-review of my own changes;
- [X] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [ ] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [X] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [X] Filters maintenance.

### Enter the issue address

 - https://github.com/AdguardTeam/AdguardFilters/issues/136570

### Add your comment and screenshots

None.

### Terms

- [X] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: BaseFilter/sections/antiadblock.txt [hunk 1]
```diff
@@ -3031,17 +3031,12 @@ beatsnoop.com#%#//scriptlet("abort-current-inline-script", "document.createEleme
...
-!#if (adguard_app_ios || adguard_ext_android_cb)
-bowfile.com#@#.adsbygoogle
-@@||ugroocuw.net/*/$domain=bowfile.com
+! https://github.com/AdguardTeam/AdguardFilters/issues/136570
 @@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=bowfile.com
-@@||cloudfront.net/?*=$domain=bowfile.com
+@@/^https?:\/\/[0-9a-z]{13,14}\.cloudfront\.net\/\?[a-z]{3,5}=\d{6}$/$script,domain=bowfile.com
+||d192r5l88wrng7.cloudfront.net^$redirect=noopframe,domain=bowfile.com
+!#if (adguard_app_ios || adguard_ext_android_cb)
+@@||d192r5l88wrng7.cloudfront.net^$subdocument,domain=bowfile.com
```
[the remark this answers is not in the document]
Comment (piquark6046) [reply BaseFilter/sections/antiadblock.txt:3039]: `@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=bowfile.com` is preserved.

[the remark this answers is not in the document]
Comment (piquark6046) [reply BaseFilter/sections/antiadblock.txt:3039]: Hmm. It seems that they updated their script.

[the remark this answers is not in the document]
Comment (AdamWr) [reply BaseFilter/sections/antiadblock.txt:3039]: On my end these rules works:
```adblock
||cloudfront.net/?rwlrd=$redirect=noopframe,domain=bowfile.com
||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$script,xmlhttprequest,redirect=googlesyndication-adsbygoogle,domain=bowfile.com,important
```

[the remark this answers is not in the document]
Comment (AdamWr) [reply BaseFilter/sections/antiadblock.txt]: Alternative solution with scriptlets:
```adblock
bowfile.com#%#//scriptlet('set-constant', 'adsbygoogle.loaded', 'true')
bowfile.com#%#//scriptlet('prevent-element-src-loading', 'iframe', 'cloudfront.net/?')
bowfile.com#%#//scriptlet('prevent-element-src-loading', 'script', 'pagead2.googlesyndication.com')
bowfile.com#%#//scriptlet('abort-current-inline-script', 'Promise', 'continue;}break;}}else')
```

**Review by AdamWr — APPROVED** [review]

**Review by zloyden — APPROVED** [review]

**Pull request merged**